appserviceenvironment-serviceplan-logicapp/readme.md

```diff
@@ -0,0 +1,41 @@
+# Deploy an App Service Environment with an AppService Plan and Logic App onto an existing Virtual Network
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fappserviceenvironment-with-serviceplan%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+Built by: [YOSSIDAHAN](https://github.com/yossidahan)
+
+This template creates a new AppService Environment on a specified Subnet in a pre-existign Virtual Netowrk. 
+It proceeds to create an AppService hosting plan preparing the environment to recieve Web, API and Logic apps.
+Finally - it deploys a sample Logic app that calls http://www.example.com on a schedule
+
+The template assumes a 'Classic' network has been pre-provisioned that includes a subnet onto which to deploy the environment.
```

Comment:
I've hopefully clarified the comment in the readme.md file - I believe that ASE requires a V1 network